docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc

```diff
@@ -13,20 +13,34 @@ You must configure your build and deployment settings to ensure that your applic
 [discrete]
 == Prerequisites
 
+* A {SpringBoot} application, such as xref:spring-boot-available-missions-and-boosters[a booster].
 * Fabric8 Maven Plugin used to deploy your application to OpenShift.
 * Spring Boot Maven Plugin used to package your application.
 
 [discrete]
 == Procedure
-// JSP pages, JSP Standard Tag Library (JSTL), ...
 
-. Define the `repackage` Maven goal for the Spring Boot Maven plugin in the `pom.xml` file of your project:
+. Add `war` packaging the `pom.xml` file of your project.
```

Comment:
Are you missing _to_ here?

I would also use a colon instead of a period at the end of this sentence, but that's subjective.

Reply:
thanks @tradej.

Fixing.